openmct.js

```diff
@@ -21,6 +21,50 @@
  *****************************************************************************/
 /*global module*/
 
+/**
+ * @typedef {object} BuildInfo
+ * @property {string} version
+ * @property {string} buildDate
+ * @property {string} revision
+ * @property {string} branch
+ */
+
+/**
+ * @typedef {object} OpenMCT
```

Comment:
This is a "hack" to get types for the OpenMCT global export. Really this should be defined as a class that extends EventEmitter, but we won't be able to specify that without moving to ES6 classes. AMD is not well-supported by the JSDoc --> TypeScript conversion tool.